buildlib/azure-pipelines-release-drp.yml

```diff
@@ -6,9 +6,18 @@ pr:
   - master
   - v*.*.x
 
+schedules:
+- cron: '0 0 * * *'
+  displayName: Daily midnight test
+  always: true
+  branches:
+    include:
+    - master
+
 variables:
   DOCKER_OPT_VOLUMES: -v /hpc/local:/hpc/local
-  REPO_MIRROR: harbor-pdc.nvidia.com
+  # REPO_MIRROR: harbor-pdc.nvidia.com
```

Comment:
remove commented out line?

Reply:
This was supposed to be a temp workaround, but as the PDC mirror is still down, let's keep using the main repo.